### PR TITLE
Fix prying speed & log

### LINKED ISSUE #27745: Prying inconsistencies
## Description

Prying for stuff like doors was made into its own system in #19394.
Tile prying originally operated on its own logic with a tileprying component/system.
It was moved into tools in #24432.

Prying logic now diverges, which leads to stuff like pryingmodifier in the prying component being unused for tile prying.

### LINKED ISSUE #28334: Jaws of Life doafter continues when not held
## Description

Jaws of Life's doafter continues even if you put away the item after starting to use it, allowing you to break open doors without visibly holding the item.

The doafter should break if the Jaws is no longer in the user's hand.

**Reproduction**

1. Start opening a door with Jaws of Life
2. Put Jaws of Life in your backpack
3. Watch doafter bar continue despite not holding the Jaws

**Additional context**

Syndicate Jaws of Life obviously have this problem too.

### PR DESCRIPTION
**Pull request by MilenVolf**
## About the PR
Fixes #28334
Fixes #27745

## Why / Balance
Prying speed was not properly applied;
Jaws of life now use reduced delay for tile prying.

## Technical details
* Deleted unused obsolete `ToolForcePoweredComponent`. Already have fields for this in `PryingComponent`;
* Replaced some obsolete methods;
* Added `NeedHand` arg with check in DoAfter to prevent tool from continuing doafter when tool's not being held;
* Fixed prying logs (`tool != null` was always true, so second admin log was never used)

## Media

https://github.com/space-wizards/space-station-14/assets/63782763/8557ec25-e583-45dc-9944-c1f924635ab7

### Logs
With tool:
image
Without tool:
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- fix: Jaws of life, xeno and spiders now use proper prying speed that will lead into a faster prying.
- tweak: Jaws of life now pries tiles much faster.

### CODE REVIEW

**Comment by Plykiya** [comment]
oh my god, prying a locked door with jaws of life isn't supposed to take like 25 seconds?

**Comment by MilenVolf** [comment]
> oh my god, prying a locked door with jaws of life isn't supposed to take like 25 seconds?

Yep)

**Comment by MilenVolf** [comment]
> also test fail is real

Working on it.
Honestly, everyhing about tools feels messy. For example, it could've checked for the component to get some data from it's fields, instead of just duplicating it in the prototype.
Maybe someday somebody will refactor it... probably next week :trollface:

File: Resources/Prototypes/Entities/Objects/Tools/jaws_of_life.yml [hunk 1]
```diff
@@ -18,15 +18,15 @@
     slots:
       - Belt
   - type: ToolTileCompatible
+    delay: 0.5
```
Comment (MilenVolf) [inline Resources/Prototypes/Entities/Objects/Tools/jaws_of_life.yml:21]: Idk if it's too fast. Let me know if the value needs to be changed or should I keep it default.

**Comment by nikthechampiongr** [comment]
> > also test fail is real
> 
> Working on it. Honestly, everyhing about tools feels messy. For example, it could've checked for the component to get some data from it's fields, instead of just duplicating it in the prototype. Maybe someday somebody will refactor it... probably next week :trollface:

Tools are very messy yea, before the pryingsystem existed, prying was based on the tool quality and was hacked into stuff like the airlock and door systems making it a pain to work with. When it was separated from the tool system I couldn't completely decouple it because it was used for stuff like construction and multi use tools so I just added a special case for the prying component. 

Tile prying for some reason even before that was its own special component and system which later kind of was integrated into the tool system. 

If you would like to deal with all of that good luck and thank you.

**Pull request merged**